win32ss/user/user32/user32.spec

```diff
@@ -744,7 +744,7 @@
 735 stdcall WindowFromPoint(double)
 736 stdcall keybd_event(long long long long)
 737 stdcall mouse_event(long long long long long)
-738 varargs wsprintfA(str str)
-739 varargs wsprintfW(wstr wstr)
+738 varargs wsprintfA(ptr str)
+739 varargs wsprintfW(ptr wstr)
```

Comment:
Why is the first parameter "ptr"? It's a pointer to string, much like the second parameter....

Reply:
It's a pointer to a buffer, i.e. it's not initialized on input. Remember that the `ptr` vs `str` distinction is for relay tracing.

Reply:
Thanks for the info!